client/context/load.go

```diff
@@ -0,0 +1,123 @@
+package context
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+
+	"github.com/docker/docker/pkg/contextstore"
+	"github.com/docker/go-connections/tlsconfig"
+	"github.com/pkg/errors"
+)
+
+// Context is a typed wrapper around a context-store context
```

Comment:
I don't fully grok the meanings/implication of this comment. It looks from the fields like it is specifically wrapping the information which an endpoint relating to a Docker Engine (rather than, say, a Kube api server) would contain?

I suppose that is implied by the package here but I think the comment could usefully be clarified to be specific on that. e.g. `Context is a typed wrapper around a context-store context describing a Docker Engine endpoint` or something.

Thinking in terms of the rendered godoc a package level overview comment might also help.

Reply:
:+1